extensions/wikia/WallNotifications/tests/WallNotificationEntityTest.php

```diff
@@ -72,50 +68,75 @@ private function getWallMessageMock( array $params ) {
 				'isEdited',
 				'getLastEditSummary',
 				'getMetaTitle',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "getMetaTitle" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)